### PR TITLE
Add `github-activity`

### PR DESCRIPTION
**Pull request by jtpio**
Needed for https://github.com/conda-forge/staged-recipes/pull/16123

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jtpio** [comment]
cc @choldgraf @consideRatio @minrk @manics @blink1073 would you like to be added as a maintainer? Thanks!

File: recipes/github-activity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+about:
+  home: https://github.com/executablebooks/github-activity
+  summary: Simple markdown changelogs for GitHub repositories
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
```
Comment (manics) [inline recipes/github-activity/meta.yaml:50]:
```suggestion
    - jtpio
    - manics
```
I'm happy to be added!

Comment (jtpio) [reply recipes/github-activity/meta.yaml:50]: Thanks!

**Comment by consideRatio** [comment]
@jtpio yes thank you!

File: recipes/github-activity/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - manics
```
Comment (consideRatio) [inline recipes/github-activity/meta.yaml:51]:
```suggestion
    - manics
    - consideratio
```

Comment (jtpio) [reply recipes/github-activity/meta.yaml:51]: Thanks!

**Comment by blink1073** [comment]
👍🏼

File: recipes/github-activity/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - consideratio
```
Comment (jtpio) [inline recipes/github-activity/meta.yaml:52]:
```suggestion
    - consideratio
    - blink1073
```

**Comment by choldgraf** [comment]
I'm happy to be listed as a maintainer, but conda forge is still a kind of magical black box to me so if there's anything non trivial that needs to happen I won't be able to do it 😬

**Comment by jtpio** [comment]
@choldgraf no problem. We already have a few maintainers now so we can move forward with this. Additional maintainers can still be added later on.

Thanks all!

**Comment by jtpio** [comment]
@conda-forge/help-python this is ready for review, thanks!

**Pull request merged**